src/miner.cpp

```diff
@@ -342,6 +342,8 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
         txNew.vout.resize(1);
         txNew.vout[0].scriptPubKey = scriptPubKeyIn;
         txNew.vout[0].nValue = GetBlockSubsidy(nHeight, chainparams.GetConsensus());
+        // Set coinbase expiry
+        txNew.nExpiryHeight = nHeight;
```

Comment:
Set to 0 since expiry height does not apply to coinbase txs.
Is there a reason to set this to nHeight?